src/components/tree/stories/daikin-tree.stories.ts

```diff
@@ -189,3 +190,24 @@ export const Selectable: Story = {
     );
   }),
 };
+
+export const Manual: Story = {
+  args: {
+    ...Default.args,
+    selectable: false,
+    selectedItems: ["1"],
```

Comment:
It would be better to indicate that multiple selections are possible.

```suggestion
    selectedItems: ["1", "2-2"],
```

Reply:
@yodas7 
Is it really necessary to go to the trouble of indicating that multiple selections are possible?
To begin with, I thought that support for multiple selections was a kind of sub-function.

I couldn't think of any use case where two trees being selected would be necessary.